src/ApiCore/GapicClientTrait.php

```diff
@@ -96,105 +97,165 @@ private static function getGapicVersion(array $options)
      *     @type string $versionFile
      *           The path to a file which contains the current version of the
      *           client.
-     *     @type string $restClientConfigPath
-     *           The path to a REST configuration file.
      *     @type string $descriptorsConfigPath
      *           The path to a descriptor configuration file.
      *     @type string $serviceName
      *           The name of the service.
      *     @type string $serviceAddress
-     *           The domain name of the API remote host.
-     *     @type mixed $port
-     *           The port on which to connect to the remote host.
-     *     @type Channel $channel
-     *           A `Channel` object. If not specified, a channel will be constructed.
-     *           NOTE: This option is only valid when utilizing the gRPC transport.
-     *     @type ChannelCredentials $sslCreds
-     *           A `ChannelCredentials` object for use with an SSL-enabled channel.
-     *           Default: a credentials object returned from
-     *           \Grpc\ChannelCredentials::createSsl().
-     *           NOTE: This option is only valid when utilizing the gRPC
-     *           transport. Also, if the $channel optional argument is
-     *           specified, then this argument is unused.
-     *     @type bool $forceNewChannel
-     *           If true, this forces gRPC to create a new channel instead of
-     *           using a persistent channel. Defaults to false.
-     *           NOTE: This option is only valid when utilizing the gRPC
-     *           transport. Also, if the $channel optional argument is
-     *           specified, then this option is unused.
-     *     @type CredentialsLoader $credentialsLoader
```

Comment:
Is it now the case users cannot provide a `credentialsLoader` instance of their own, unless they create their own `Transport` without using the factory?